### PR TITLE
tr-ex.me

### LINKED ISSUE #134088: tr-ex.me
### Issue URL (Ads)
 [https://tr-ex.me/%E7%BF%BB%E8%A8%B3/%E8%8B%B1%E8%AA%9E-%E6%97%A5%E6%9C%AC%E8%AA%9E/news+releases](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftr-ex.me%2F%25E7%25BF%25BB%25E8%25A8%25B3%2F%25E8%258B%25B1%25E8%25AA%259E-%25E6%2597%25A5%25E6%259C%25AC%25E8%25AA%259E%2Fnews%2Breleases)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.3.3
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 Fix #134088 

### Add your comment and screenshots

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: MobileFilter/sections/specific_web.txt [hunk 1]
```diff
@@ -48,7 +48,7 @@ wired.com,sports.ru,manager-magazin.de,amp.macwelt.de,amp.fitforfun.de,tomsguide
 ! ##.inline-ad-container
 scmp.com,theinventory.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,splinternews.com,theinventory.com,theonion.com,theroot.com,thetakeout.com##.inline-ad-container
 ! ##.ads
-berkem.ru,pixiv.net,cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m.ura.news,up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org,m-economictimes-com.cdn.ampproject.org,m.winfuture.de##.ads
+tr-ex.me,berkem.ru,pixiv.net,cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m.ura.news,up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org,m-economictimes-com.cdn.ampproject.org,m.winfuture.de##.ads
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:51]: It's in desktop site too tho not as big placeholder, so move to Base.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: MobileFilter/sections/specific_web.txt (hunk 1 above)
Comment (SKEIDs) [reply MobileFilter/sections/specific_web.txt:51]: I see.

**Review by Yuki2718 — APPROVED** [review]

File: MobileFilter/sections/specific_web.txt [hunk 2]
```diff
@@ -48,7 +48,6 @@ wired.com,sports.ru,manager-magazin.de,amp.macwelt.de,amp.fitforfun.de,tomsguide
 ! ##.inline-ad-container
 scmp.com,theinventory.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,splinternews.com,theinventory.com,theonion.com,theroot.com,thetakeout.com##.inline-ad-container
 ! ##.ads
-berkem.ru,pixiv.net,cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m.ura.news,up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org,m-economictimes-com.cdn.ampproject.org,m.winfuture.de##.ads
```
Comment (zloyden) [inline MobileFilter/sections/specific_web.txt:51]: Why do you remove this?
At least is needed for `m.ura.news`.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

File: MobileFilter/sections/specific_web.txt (hunk 2 above)
Comment (SKEIDs) [reply MobileFilter/sections/specific_web.txt]: sorry. I missed it. and thank you for fix.

**Pull request merged**